cpp/CMakeLists.txt

```diff
@@ -744,3 +745,79 @@ add_custom_command(OUTPUT CUDF_DOXYGEN
                    VERBATIM)
 
 add_custom_target(docs_cudf DEPENDS CUDF_DOXYGEN)
+
+
+####################################################################################################
+# - cuDatasource - OFF by default due to increased number of dependencies
```

Comment:
```suggestion
# - cudf_kafka - OFF by default due to increased number of dependencies
```

Reply:
thanks for catching, had simply overlooked this one